src/main/java/ee/tuleva/onboarding/auth/AccessAndRefreshToken.java

```diff
@@ -0,0 +1,7 @@
+package ee.tuleva.onboarding.auth;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record AccessAndRefreshToken(
```

Comment:
"And" in a class name (or method name) is a code smell - it hints at a Single Responsibility violation ("and" means, by definition, that the class does two things, not one). I would rename this to just "Token", "AuthToken", "TokenPair" or something simple like that.